### PR TITLE
[AV-1936] add go install instruction to README

### PR DESCRIPTION
**Pull request by felipemadero**
Closes https://github.com/ava-labs/avalanche-network-runner/issues/184

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,21 +14,59 @@ This tool may be especially useful for development and testing.
...
+
+If a particular version is required, change the release at the end of the command, eg:
+
+```sh
+go install github.com/ava-labs/avalanche-network-runner/cmd/avalanche-network-runner@v1.1.3
+```
+
+### Install by release download
+
+Does not requires go lang to be installed on the system.
```
Comment (holisticode) [inline README.md:36]: same

File: README.md [hunk 2]
```diff
@@ -14,21 +14,59 @@ This tool may be especially useful for development and testing.
 
 ## Installation
 
-### Download
+### Install using go tool
+
+This is the preferred way to install. Requires go lang to be installed on the system.
```
[the remark this answers is not in the document]
Comment (holisticode) [reply README.md:19]: Should we add some link about how to install `go` or should this be considered up to the user?

[the remark this answers is not in the document]
Comment (felipemadero) [reply README.md:19]: added a link to main golang page install instructions

File: README.md [hunk 3]
```diff
@@ -33,31 +36,61 @@ export PATH=$PWD:$PATH
...
+#### Installing in Custom Location
 
 To download the binary into a specific directory, run:
 
 ```
 curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-network-runner/main/scripts/install.sh | sh -s -- -b <relative directory>
 ```
 
-## Running from source
+### Install using go tool
```
Comment (dasconnor) [inline README.md:47]: `using golang`

**Review by dasconnor — APPROVED** [review]
one small nit

**Pull request merged**